Content/Tiles/Vitric/Temple/TempleDecoration.cs

```diff
@@ -0,0 +1,210 @@
+﻿using StarlightRiver.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Terraria.DataStructures;
+using Terraria.Enums;
+using StarlightRiver.Core.Loaders;
+using Terraria.ModLoader;
+using Terraria.ID;
+using Terraria.ObjectData;
+using Terraria;
+
+namespace StarlightRiver.Content.Tiles.Vitric.TempleDecoration
+{
+	public class TempleDecoration : SimpleTileLoader
+	{
+		//comment out
+		//private static AnchorData anchor = new AnchorData(AnchorType.SolidTile | AnchorType.SolidWithTop | AnchorType.Table, 2, 0);
+		//private static FurnitureLoadData boxData = new FurnitureLoadData(2, 2, 0, SoundID.Dig, true, new Color(255, 200, 100), false, false, "Music Box", anchor);
+
+		public override string AssetRoot => "StarlightRiver/Assets/Tiles/Vitric/TempleDecoration/";
+
+		public override void Load()
+		{
+			//todo dust
+			AnchorData anchorTabletop1X = new AnchorData(AnchorType.SolidTile | AnchorType.SolidWithTop | AnchorType.Table, 1, 0);
+
+			LoadDecoTile("TempleCandle", "Temple Candle",
+				new FurnitureLoadData(1, 1, 0, SoundID.Dig, false, new Color(140, 97, 86), false, false, "Candle", anchorTabletop1X));
+
+			LoadDecoTile("TempleCandleCrystal", "Temple Crystal Candle",
+				new FurnitureLoadData(1, 2, 0, SoundID.Dig, false, new Color(80, 131, 142), false, false, "Candle", anchorTabletop1X));
+
+
+			AnchorData anchorFloor1X = new AnchorData(AnchorType.SolidTile | AnchorType.SolidWithTop, 1, 0);
```

Comment:
Might be smart to define all of the anchors at the top and then call all of the loads?